pkg/internal/apis/certmanager/validation/issuer_test.go

```diff
@@ -605,11 +601,7 @@ func TestValidateACMEIssuerDNS01Config(t *testing.T) {
 				},
 			},
 			errs: []*field.Error{
-				field.Required(fldPath.Child("azuredns", "clientSecretSecretRef", "name"), "secret name is required"),
-				field.Required(fldPath.Child("azuredns", "clientSecretSecretRef", "key"), "secret key is required"),
-				field.Required(fldPath.Child("azuredns", "clientID"), ""),
 				field.Required(fldPath.Child("azuredns", "subscriptionID"), ""),
-				field.Required(fldPath.Child("azuredns", "tenantID"), ""),
```

Comment:
Please also add a few additional test cases to cover the new fields/field changes just added - specifically we should exercise the entirety of the block of code you've added into the validations :)